helm/kubernetes-cluster-prep/utils.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
What would be the best way to share this utility file between chart directories?